src/cladetime/assign_clades.py

```diff
@@ -3,141 +3,114 @@
+    # https://docs.nextstrain.org/projects/ncov/en/latest/reference/metadata-fields.html
+    cols = [
+        "clade_nextstrain",
+        "country",
+        "date",
+        "division",
+        "genbank_accession",
+        "genbank_accession_rev",
+        "host",
+    ]
 
+    # clean and filter metadata (same process used to generate the weekly clade list)
+    filtered_metadata = filter_covid_genome_metadata(metadata, cols)
 
-def get_sequence_metadata(config: Config):
-    """Generate tabular representation of the downloaded genbank sequences."""
+    # add filters based on user input
+    filtered_metadata = filtered_metadata.filter(pl.col("date") >= sequence_collection_date)
 
-    fields = "accession,sourcedb,sra-accs,isolate-lineage,geo-region,geo-location,isolate-collection-date,release-date,update-date,virus-pangolin,length,host-name,isolate-lineage-source,biosample-acc,completeness,lab-host,submitter-names,submitter-affiliation,submitter-country"
+    return filtered_metadata
 
-    with open(config.ncbi_sequence_metadata_file, "w") as f:
-        subprocess.run(
-            [
-                "dataformat",
-                "tsv",
-                "virus-genome",
-                "--inputfile",
-                f"{config.data_path}/ncbi_dataset/data/data_report.jsonl",
-                "--fields",
-                f"{fields}",
-            ],
-            stdout=f,
-        )
 
-    logger.info("extracted sequence metadata", metadata_file=config.ncbi_sequence_metadata_file)
+def filter_sequences(filtered_metadata: pl.LazyFrame, full_sequence_file: Path, tmpdir: Path) -> Path:
+    """Create input sequence file for clade assignment."""
 
+    # This is where we are going to use biopython to filter the sequence file to a smaller version
+    # that only includes sequences in the filtered metadata.
+    # Current thinking is that the sequence filtering itself will be a CladeTime method called here
+    return Path.home()
 
-def assign_clades(config: Config, nextclade_dataset_path: str):
+
+def assign_clades(tree_path: Path, sequence_path: Path):
     """Assign downloaded genbank sequences to a clade."""
 
+    # FIXME: restore the nextclade run invocation once we've refactored
+    # the code for creating its inputs.
+    # The code below that actually invokes the nextclade will likely move to a CladeTime method
     subprocess.run(
         [
             "nextclade",
-            "run",
-            f"{config.ncbi_sequence_file}",
-            "--input-dataset",
-            nextclade_dataset_path,
-            "--output-csv",
-            f"{config.assignment_no_metadata_file}",
+            "--version",
         ]
     )
 
-    logger.info("Assigned sequences to clades via Nextclade CLI", output_file=config.assignment_no_metadata_file)
+    logger.info("Assigned sequences to clades via Nextclade CLI", output_file="some path stuff")
```

Comment:
A temporary placeholder so the logger doesn't error when referencing a config attribute that no longer exists.